test/test_sandbox.py

```diff
@@ -39,5 +39,12 @@ def test_sandbox():
     assert bob.search(["#iris_dataset"])  # noqa: F821
     assert bob.search(["#wine_dataset"])  # noqa: F821
     assert bob.search(["#linnerrud_dataset"])  # noqa: F821
-    assert bob.search(["#fashion_mnist"])  # noqa: F821
-    assert bob.search(["#cifar10"])  # noqa: F821
+
+    try:
+        from tensorflow.keras import datasets
+
+        foo = datasets.__version__  # to overcome the Lint error of imported but not used
```

Comment:
please add instead of this, at the end of  the import line : ` # pragma: no cover`

Reply:
i've done it ;)